tests/test_delete.py

```diff
@@ -0,0 +1,35 @@
+import utilities.methods
+from data.testing_data import *
+import pytest
+
+session = utilities.methods.create_session()
+
+
+@pytest.mark.parametrize("booker", bookers)
+def test_delete_booking(booker):
+    """Create booking"""
+    results2 = utilities.methods.create_booking(booker, session)
+    create_booking_response = results2[0]
+    create_booking_response_json = results2[1]
+    booking_id = results2[2]
+    body = results2[3]
+
+    assert create_booking_response.status_code == 200
+    assert create_booking_response_json['booking'] == body
```

Comment:
Tutaj myślałam żeby zostawić tylko asercję na sprawdzenie statusu, bo podobnie robiłam w testach API w Postmanie. Tworzenie tego obiektu to przygotowywanie danych testowych właściwy test to usunięcie i to chce sprawdzić, ale lubię mieć informację, że test zakończył się niepowodzeniem, bo był problem z utworzeniem obiektu, a nie jego usunięciem